AutomatedTesting/Gem/PythonTests/Physics/tests/EntityComponentTests/PhysX_Collider_Component_CRUD.py

```diff
@@ -0,0 +1,127 @@
+    physx_material = os.path.join("physx", "glass.physxmaterial")
+    component_name = "PhysX Collider"
+
+    TestHelper.init_idle()
+    TestHelper.open_level("", "Base")
+
+    with Tracer() as section_tracer:
+    # 1 ) Add an Entity to manipulate
+        phsyx_collider_entity = EditorEntity.create_editor_entity("TestEntity")
+
+    # 2) Add a PhysX Collider Component to Entity
+        physx_collider = PhysxCollider(phsyx_collider_entity)
+
+    # 3) Set Box Shape and Child Properties
+        physx_collider.set_box_shape()
+
+        validate_vector3_property("Box Dimensions", physx_collider.get_box_dimensions, physx_collider.set_box_dimensions, component_name, VECTOR3_TESTS_NEGATIVE_EXPECT_PASS)
+
+    # 4) Set Capsule Shape and Child Properties
+        physx_collider.set_capsule_shape()
+
+        validate_float_property("Capsule Height", physx_collider.get_capsule_height, physx_collider.set_capsule_height, component_name, FLOAT_HEIGHT_TESTS)
+        validate_float_property("Capsule Radius", physx_collider.get_capsule_radius, physx_collider.set_capsule_radius, physx_collider.component.get_component_name(), COLLIDER_RADIUS_TESTS)
+
+    # 5) Set Cylinder Shape and Child Properties
+        physx_collider.set_cylinder_shape()
+
+        validate_float_property("Cylinder Height", physx_collider.get_cylinder_height, physx_collider.set_cylinder_height, component_name,  CYLINDER_HEIGHT_TESTS)
+        validate_float_property("Cylinder Radius", physx_collider.get_cylinder_radius, physx_collider.set_cylinder_radius, component_name, COLLIDER_RADIUS_TESTS)
+
+        validate_integer_property("Cylinder Subdivision", physx_collider.get_cylinder_subdivision, physx_collider.set_cylinder_subdivision, component_name, CYLINDER_SUBDIVISION_TESTS)
+
+    # 6) Set Sphere Shape and Child Properties
+        physx_collider.set_sphere_shape()
+
+        validate_float_property("Sphere Radius", physx_collider.get_sphere_radius, physx_collider.set_sphere_radius, component_name, COLLIDER_RADIUS_TESTS)
+
+    # 7) Set General Properties
+        validate_property_switch_toggle("Is Trigger", physx_collider.get_is_trigger, physx_collider.set_is_trigger, component_name)
+        validate_property_switch_toggle("Is Simulated", physx_collider.get_is_simulated, physx_collider.set_is_simulated, component_name)
+        validate_property_switch_toggle("In Scene Queries", physx_collider.get_in_scene_queries, physx_collider.set_in_scene_queries, component_name)
+
+        validate_vector3_property("Offset", physx_collider.get_offset, physx_collider.set_offset, component_name, VECTOR3_TESTS_NEGATIVE_EXPECT_PASS)
+        # TODO: Create Validate Quaternian Backlog: validate_vector3_property(physx_collider.get_rotation, physx_collider.set_rotation, physx_collider.component.get_component_name(), "Rotation", VECTOR3_TESTS_NEGATIVE_EXPECT_FAIL)
+
+        # o3de/o3de#12634 - For some reason I can't get this to work. Says it takes an AZStd::string,
```

Comment:
update: Would be good to follow-up about this and in the short-term remove the commented out code and add to the GHI to track and fix

Reply:
Removed the non-working code and made notes in the GHI.